src/source/index.rst

```diff
@@ -48,10 +48,6 @@ History
 
 ..  role:: strike
 
-* v.0.114 - 6 Jan 2020 - tw
```

Comment:
make new version 0.115, saying this is undone and now always like that

Reply:
done & done